test/test_parsy.py

```diff
@@ -1,4 +1,4 @@
-from parsy import string, regex, generate, ParseError, letter, digit
+from parsy import *
```

Comment:
This introduces tons of flake8 errors/warnings, plus can give static analysers/"goto definition" a harder time for anyone working on the code. I don't think it is worth it.